scripts/rollup/build.js

```diff
@@ -224,6 +228,11 @@ function getFormat(bundleType) {
       return `cjs`;
     case NODE_ESM:
       return `es`;
+    case IIFE_DEV:
```

Comment:
I don't think it's worth having a dev build of this. It doesn't have any warnings or anything. Let's just do prod only.